src/views/Records/Record.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    class="standard container-fluid"
+  >
+    <div v-if="error"
+      class="error"
+    >
+      {{ error }}
+    </div>
+
+    <div
+      v-if="!error && queryTriggered"
+      class="row"
+    >
+      <div class="col-10">
+        <span>{{ currentRecord['fairsharingRecord'].name }}</span>
+        <div class="card">
+          <div
+            v-for="(field, label, index) in currentRecord['fairsharingRecord']"
+            :key="index"
+          >
+            <b>{{ label }}: </b> {{ field }}
+          </div>
+        </div>
+      </div>
+
+      <div class="col-2">
+        <button @click="getHistory()"
+                type="button"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Attribute "type" should go before "@click".](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4965485)